hr_attendance_rest_time_included/models/hr_attendance_rest_time.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Tecnativa - Eduardo Ezerouali
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models
+
+
+class HrAttendanceRestTime(models.Model):
+    _name = "hr.attendance.rest_time"
+    _inherit = "hr.attendance"
```

Comment:
For indicating the intention of inheritance with new model, better to put before `_inherit` and then `_name`:

```suggestion
    _inherit = "hr.attendance"
    _name = "hr.attendance.rest_time"
```